components/style/values/animated/mod.rs

```diff
@@ -9,15 +9,35 @@
 //! module's raison d'être is to ultimately contain all these types.
 
 use app_units::Au;
+use std::cmp;
 use values::computed::Angle as ComputedAngle;
+use values::computed::BorderCornerRadius as ComputedBorderCornerRadius;
```

Comment:
Please no, make a submodule `border` just like there is one in `computed` and `specified`.